static/js/Misc.jsx

```diff
@@ -59,6 +62,35 @@ const __filterChildrenByLanguage = (children, language) => {
   let newChildren = chlArr.filter(x=> x.type == currLangComponent);
   return newChildren;
 };
+const GDocAdvertText = () => {
+    return    <InterfaceText>
+                <EnglishText> Add texts directly to your Google Docs with our <span id="newExtension">new extension</span>! <a href="https://sefaria.org/sheets/529099?origin=AddToSheetsPromo">Learn more</a></EnglishText>
+                <HebrewText> הוסיפו טקסטים מספריא ישירות לקובץ גוגל עם <span id="newExtension">התוסף החדש</span> שלנו! <a href="https://sefaria.org/sheets/529099?origin=AddToSheetsPromo">למדו עוד</a></HebrewText>
+             </InterfaceText>;
+}
+const GDocAdvertBox = () => {
+    const firstRender = useRef(true);
+    useEffect(() => {
+        if (firstRender.current) {
+          firstRender.current = false;
+          if (!cookie(gdocInstalled)) { // on first render and "Install Now" has never been clicked before
+              trackSidebarAdImpression(gdocsCampaign);
+          }
+        }
+      })
+    const gdocsCampaign = {campaignId: "GDocs_Promo_AddToSheet"};
```

Comment:
consider having at the top of the component CAMPAIGN_ID and CAMPAIGN_LINK so that the code is cleaner later on